onnx/onnx-ml.proto

```diff
@@ -173,9 +179,13 @@ message NodeProto {
 // a parameterized computation graph against a set of named operators that are
 // defined independently from the graph.
 message ModelProto {
+  // 'ONXF' (reversed as 46 58 4E 4F) 0x46584E4F
+  // This field MUST be present.
+  optional fixed32 magic_prefix = 1;
```

Comment:
so if I'm not mistaken it's no backward compatible change: https://developers.google.com/protocol-buffers/docs/proto - fixed32 and int64 are not interchangeable

did you try to see what happens to existing models?

Reply:
Correct - I mentioned this would break existing files, as well as other implementations that don't pick up the new definition (as they will misidentify the field). But everything is stamped with major version of zero today, so implementations will start rejecting that soon one way or another.